src/applications/simple-forms/simple-forms-patterns/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Simple Forms Patterns",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "simple-forms-patterns",
+  "rootUrl": "/simple-forms/simple-forms-patterns",
```

Comment:
The only comment I have is that you may want to indicate that this is a dummy form somehow. Like maybe make the `entryName` `mock-simple-forms-patterns` and change the URL too. Just a though to make it clear that it's not a live form. Though if you do change the `entryName`, you'll have to to change all of its references too